### PR TITLE
Some italian proverbs

### PR DESCRIPTION
**Pull request by Maaarcocr**
EDIT: I tried to use literal translation for most of them, instead of using a similar proverb in english.

### CODE REVIEW

**Comment by namin** [comment]
Good call on literal translation.

Can you add my favorite please!

chi va piano, va sano et lontano

:)

**Comment by Maaarcocr** [comment]
Fatto! 😃

File: italian.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+- a buon intenditore poche parole
+   - few words for the expert
+- la notte porta consiglio
+   - Night brings advice
+- la pazienza é la virtú dei forti
+   - patience is strong people's virtue
+- si stava meglio quando si stava peggio
+   - it was better when it was worse
+- chi va piano va sano e lontano
+   - they who go slow, go far and safe
```
Comment (namin) [inline italian.md]: they who go slow, go safe and far?

Comment (Maaarcocr) [reply italian.md]: For some reason my brain inverted the words 😮

File: italian.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+- chi fa per se fa per tre
+   - do it yourself, it's 3 times better
+- la gatta frettolosa fa i gattini cechi
+   - The hasty cat makes blind kittens 
+- chi dorme non piglia pesci
+   - the sleeper doesn't catch fishes
+- tanto va la gatta al lardo che ci lascia lo zampino
+   - the cat that tries too hard to reach the lard will leave behind its footprint
+- a caval donato non guardare in bocca
+   - don't look in the mouth of a gifted horse
```
Comment (namin) [inline italian.md:10]: haha

**Review by namin — APPROVED** [review]
I love this list, thank you. Would you mind fixing the transposition of safe and fast? Also, is it safe or sane?

**Comment by namin** [comment]
Thank you :)

**Pull request merged**